### PR TITLE
Feature/68/ci tests workflow

### PR DESCRIPTION
**Pull request by gluhovSemen**
Test running action added

### CODE REVIEW

File: tests/dummy_test.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# TODO this file will be deleted when actual tests will be ready.
+
+from manage import main
```
Comment (egorvavilov) [inline tests/dummy_test.py]: Lets create dummy test from django
https://docs.djangoproject.com/en/4.2/topics/testing/overview/

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v3
```
Comment (JedrzejMaluszczak) [inline .github/workflows/main.yaml]: why not v4?

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: Set up python
+        id: setup-python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.12'
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
```
Comment (JedrzejMaluszczak) [inline .github/workflows/main.yaml:30]: I thnik we should pass a poetry version

File: .github/workflows/main.yaml [hunk 4]
```diff
@@ -5,6 +5,17 @@ on: [pull_request, push]
 jobs:
   test:
     runs-on: ubuntu-latest
+
+    services:
+      postgres_main:
+        image: postgres:12
```
Comment (egorvavilov) [inline .github/workflows/main.yaml]: maybe let's use 14?
12 was released in 2019;)

File: tests/dummy_test.py (hunk 1 above)
Comment (gluhovSemen) [reply tests/dummy_test.py]: done

File: .github/workflows/main.yaml (hunk 2 above)
Comment (gluhovSemen) [reply .github/workflows/main.yaml]: you are right, fixed

File: .github/workflows/main.yaml (hunk 3 above)
Comment (gluhovSemen) [reply .github/workflows/main.yaml:30]: done

File: .github/workflows/main.yaml (hunk 4 above)
Comment (gluhovSemen) [reply .github/workflows/main.yaml]: changed

**Pull request merged**